src/networking.c

```diff
@@ -3440,6 +3485,22 @@ sds getAllClientsInfoString(int type, int hide_user_data) {
     return o;
 }
 
+sds getAllFilteredClientsInfoString(clientFilter *client_filter, int hide_user_data) {
+    listNode *ln;
+    listIter li;
+    client *client;
+    sds o = sdsnewlen(SDS_NOINIT, 500);
```

Comment:
Why did we choose this magic number `500` ?

Reply:
lmk folks if there is a better / right value for it. Not sure how to measure this exactly.

Reply:
Ah okay. never bothered to expand all the resolved conversation. I think we should be fine to let it expand on it's and not define this hardcoded value. WDYT?

Reply:
Yeah it would be fine to start from empty and let it expand dynamically. I don't mind. It will expand to ~300 for the first client, then double the allocation when needed.

Reply:
noted. have updated it to empty.